src/main/scala/org/constellation/domain/cluster/ClusterStorageAlgebra.scala

```diff
@@ -24,4 +25,28 @@ trait ClusterStorageAlgebra[F[_]] {
 
   def setNodeState(id: Id, state: NodeState): F[Unit]
 
+  def getActivePeersIds(withSelfId: Boolean = false): F[Set[Id]]
+  def getActivePeers: F[Map[Id, PeerData]]
+  def getActiveLightPeersIds(withSelfId: Boolean = false): F[Set[Id]]
+  def getActiveLightPeers(): F[Map[Id, PeerData]]
```

Comment:
`getActiveLightPeers` instead of `getActiveLightPeers()` to be consistent with the other methods. if it makes an action, then `()`. if not and it is only a get, no `()`

Reply:
It's exactly what happens, getActiveLightPeers() performs an action.

Reply:
what? how?

Reply:
You can look it up, but essentially what it does is it takes the list of lightPeersIds, and fetches their PeerData from the list of all peers. There is some more logic happening for this case and I'm not sure it should stay their inside a storage, but that's how I rebased it initially to test if the implementation still works. The thing is this function is needed in different places around the code base, it was previously in Cluster class, now it's in storage and it kind of clashes with the idea that storage should only store data so possibly I will need to improve it.

Reply:
but all in all, it's `get active light peers`. doesn't matter how it works under the hood. if it takes it from the database, API or Mars ;)

Reply:
Yeah kind of makes sense ;). I remember we discussed that only for a simple getter it should be parameterless (just getting the value) but here much more happens actually. Though it's true that behind the interface it's not distinguishable. That said I can change it to parameterless and we can assume that every get function with no parameters should be in parameterless notation.